docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +29,76 @@ hosts_additional_entries:
 
-Two certificate files are required to use TLS securely with authentication,
-which will be provided by your Certificate Authority:
+## General procedure
 
-* the server certificate with private key
-* the CA certificate with any intermediate certificates
+To enable TLS encryption the following steps are needed.
+
+1. Acticate tls encryption for both endpoints
+
+  * To enable external TLS encryption:
+
+    ```yaml title="environments/kolla/configuration.yml"
+    kolla_enable_tls_external: "yes"
+    ```
+  * To enable internal TLS encryption:
+
+    ```yaml title="environments/kolla/configuration.yml"
+    kolla_enable_tls_internal: "yes"
+     ```
+
+2. Add the combined server certificate and private key to the following locations in the configuration repository:
+  * private key & certificates for `kolla_external_fqdn`: `environments/kolla/certificates/haproxy.pem`
+  * private key & certificates for `kolla_internal_fqdn`: `environments/kolla/certificates/haproxy-internal.pem`
+3. Encrypt the certificates using ansible vault:
+   ```
+   make ansible_vault_edit FILE=environments/kolla/certificates/haproxy.pem
+   make ansible_vault_edit FILE=environments/kolla/certificates/haproxy-internal.pem
+   ```
+4. Add the changesto the Git repository
+   ```
+   git add environments/kolla/certificates/haproxy.pem \
+     environments/kolla/certificates/haproxy-internal.pem \
+     environments/kolla/configuration.yml
 
-The combined server certificate and private key needs to be provided at
-the following locations in the configuration repository:
+   git commit -m "Add new certificates" environments/kolla/certificates/haproxy.pem \
+     environments/kolla/certificates/haproxy-internal.pem \
+     environments/kolla/configuration.yml
+   ```
+5. Rollout changes
+   ```
+   osism apply loadbalancer
+   ```
 
-* private key & certificates for `kolla_external_fqdn`: `environments/kolla/certificates/haproxy.pem`
-* private key & certificates for `kolla_internal_fqdn`: `environments/kolla/certificates/haproxy-internal.pem`
+### Self-signed certificates
 
-## Generating TLS certificates with Let’s Encrypt
+OSISM supports the usage of self-signed certificates with a custom CA i.e if you
+are running a test installation or for interim purposes.
 
-## Self-signed certificates
+Two certificate files are required to use TLS securely with authentication,
+which will be provided by your custom Certificate Authority:
 
-The use of self-signed certificates with a custom CA is possible. However, a few
-additional parameters are then required in the configuration so that the custom CA
-is known everywhere and the self-signed certificates are accepted as valid.
+* the server certificate with private key
+* the CA certificate with any intermediate certificates
+
+The following procedure describes the prepration procedure for the CA, which is later followed
+by the general steps described above.
```

Comment:
Please be specific to which steps this is referring to. I could not tell as a user.